### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S contrib,target -L crate,asser,te`

### CODE REVIEW

**Comment by fabricereix** [comment]
Thanks a lot @kianmeng.
This is very good, we could maybe add this tool in the CI.
Relating to the changes, I would  prefer to have distinct PRs for doc and for code.
But it might not be easy to update this PR. What do you think @jcamiel ?

**Comment by jcamiel** [comment]
I think it's ok (it just doc or comment) apart for the typo in `base64.rs` that I think will not pass the unit test. We can merge it as it (provided the test units are ok) because it will be difficult to split between docs and comments.

**Comment by kianmeng** [comment]
@fabricereix @jcamiel 

Let me know if you guys prefer which tool for regression testing in the CI:

- https://github.com/crate-ci/typos
- https://github.com/codespell-project/codespell

I'll create another PR for this.

File: packages/hurl_core/src/parser/base64.rs [hunk 1]
```diff
@@ -32,7 +32,7 @@ https://en.wikipedia.org/wiki/Base64
 Test padding/no-padding
 
 Encoded
-YW55IGNhcm5hbCBwbGVhcw==		any carnal pleas   # [97, 110, 121, 32, 99, 97, 114, 110, 97, 108, 32, 112, 108, 101, 97, 115]
```
Comment (jcamiel) [inline packages/hurl_core/src/parser/base64.rs:35]: `any carnal pleas` should be left as it as it's the base64 decoded string of  `YW55IGNhcm5hbCBwbGVhcw==`

File: packages/hurl_core/src/parser/base64.rs [hunk 2]
```diff
@@ -236,7 +236,7 @@ mod tests {
     fn test_decode_with_padding() {
         let mut reader = Reader::init("YW55IGNhcm5hbCBwbGVhcw==;");
         let decoded = parse(&mut reader);
-        assert_eq!(decoded, b"any carnal pleas");
```
Comment (jcamiel) [inline packages/hurl_core/src/parser/base64.rs:239]: Same remark as previous, `any carnal pleas` should be left for the unit test to pass

File: packages/hurl_core/src/parser/base64.rs [hunk 3]
```diff
@@ -248,7 +248,7 @@ mod tests {
     #[test]
     fn test_decode_without_padding() {
         let mut reader = Reader::init("YW55IGNhcm5hbCBwbGVhcw;");
-        assert_eq!(parse(&mut reader), b"any carnal pleas");
```
Comment (jcamiel) [inline packages/hurl_core/src/parser/base64.rs:251]: `any carnal pleas` should be left for the unit test to pass

**Review by jcamiel — COMMENTED** [review]
oK everything is good now!

**Review by jcamiel — APPROVED** [review]

**Comment by jcamiel** [comment]
@kianmeng Hi, I'm sorry, would it be possible to update your PR without the change in `.github/workflows/accept-pull-request.yml`? Out CI doesn't allow for changes in workflow, I'm going to look at it after but I want first to merge your PR asap, 

Many thanks!

**Comment by kianmeng** [comment]
@jcamiel No problem. :smile: 

I've reverted the change. Please check again.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Comment by fabricereix** [comment]
@kianmeng  
not sure about which one between `typos` and `codespell` is more relevant, or if there are redundant.
we can always give a try for both of them

**Pull request merged**